memote/support/biomass.py

```diff
@@ -176,14 +180,56 @@ def find_direct_metabolites(model, reaction):
         that are taken up to be consumed by the biomass reaction only.
 
     """
-    transport_reactions = set(helpers.find_transport_reactions(model))
-    exchange_reactions = set(model.exchanges)
-    biomass_reactions = set(helpers.find_biomass_reaction(model))
-
-    combined_set = transport_reactions | exchange_reactions | biomass_reactions
+    biomass_rxns = set(helpers.find_biomass_reaction(model))
+    tra_bou_bio_rxns = helpers.find_tra_bou_bio_reactions(model, biomass_rxns)
```

Comment:
That's a terrible function name btw :stuck_out_tongue:

Reply:
Transport/Boundary/Biomass reactions :D I thought it was brilliant!

Reply:
The reason I passed in the argument was to remove unnecessary/redundant calculations. Both this function and the function it called (`find_tra_bou_bio_reactions`) call the same function (`find_biomass_reaction`). So in order to prevent the function being called twice, I added in an optional, second parameter so the biomass reaction would only be calculated once.